### ISSUE TITLE
Failed to read: EISDIR, read Use

### ISSUE THREAD

**Issue by kevinohara80** [Two-Screen/grunt-zipstream#3]
I'm getting this error when trying to zip a tmp directory. I am currently running grunt 0.4:  

```
$ grunt --verbose
...
Running "zip:build" (zip) task
Verifying property zip.build exists in config...OK
Files: tmp -> build/staticresources/mytest.zip
Opening build/staticresources/mytest.zip...OK
Zipping tmp...ERROR
Warning: Failed to read: EISDIR, read Use --force to continue.
```

Here is the zip portion of my Gruntfile:

``` js
zip: {
  build: {
    src: ['tmp'],
    dest: 'build/staticresources/<%= pkg.name %>.zip',
  }
}
```

The zip is being created but nothing is being written to it. My tmp directory contains one file in another directory like so: 

```
tmp/js/myfile.min.js
```

Any ideas?

**Comment by stephank** [comment]
I published 0.2.1 on NPM with a fix for this. Thanks for reporting!

**Comment by kevinohara80** [comment]
Thanks. I'll update today!

On Thursday, February 21, 2013, Stéphan Kochen wrote:

> I published 0.2.1 on NPM with a fix for this. Thanks for reporting!
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Two-Screen/grunt-zipstream/issues/3#issuecomment-13880221.

## 

_Kevin O'Hara_ | gv:248.906.8538 |
tw:kevino80http://twitter.com/#!/kevino80| github:
kevinohara80 https://github.com/kevinohara80
